mqttclients/ts/mqttjsClientExtensions/src/utf8JsonSerializer.ts

```diff
@@ -0,0 +1,13 @@
+import { IMessageSerializer } from ".";
+
+export class Utf8JsonSerializer implements IMessageSerializer {
+    public contentType = 'application/json';
+
+    public fromBytes<T>(payload: Buffer): T {
+        return JSON.parse(payload.toString('utf8')) as T;
+    }
+
+    public toBytes<T>(payload: T): Buffer {
+        return Buffer.from(JSON.stringify(payload), 'utf8');
```

Comment:
do we need to deserialze? I thought we could use the underlying stream to initialize the Buffer